Test large file action

kuritz opened the pull request:
*Please title your pull request in this format: The event name and year in the title of the PR, along with a description of what is being changed, i.e., `[CHI-2019] Add Bluth Company as a sponsor`*

If you are adding or removing organisers, please email info@devopsdays.org with the details of who is being added and removed, along with the full names and email addresses of the new team, so we can update Slack and the mailing list.

Up to date guidelines for pull requests can be found in the [contributing guide](../CONTRIBUTING.md#guidelines).

mattstratton commented:
This looks good to me! We will have to see how it works overall I’m sure. 

Would love to get a 👍 from @yvovandoorn as well

sebagomez committed:
Include the LonkedIn profile url in the add_speaker script

Signed-off-by: Sebastián Gómez <sebastiangomezcorrea@gmail.com>

sebagomez committed:
Include the LonkedIn profile url in the add_speaker script

Signed-off-by: Sebastián Gómez <sebastiangomezcorrea@gmail.com>

sebagomez committed:
Update add_speakers.sh

removed linked in entry

sebagomez committed:
Update add_speakers.sh

removed linked in entry

Commit:
[MSP-2024] Add 2024 Minneapolis sponsorship prospectus, Open CFP, and Ticket Sales

glasnt committed:
Merge pull request #13553 from omargohan/feature/sponsor_rebrand_logo

[LA-2024] Solarwinds logo update request

glasnt committed:
Merge pull request #13550 from devopsdaysbham/main

[BHM-AL-2024] Organizer updates

glasnt committed:
Merge pull request #13525 from DodAracaju/Aju-2024/creating_event

[AJU-2024] publishing files for devopsdays aracaju

DaniloBNascimento committed:
adding new organizer Jodemi Cruz Santos (#13554)

jrn00 committed:
[RDU-2024] Added Mel and Jason as organizers and removed Alex.  Also removed links to CFP (#13552)

* # out cfp_link

* added Mel Cone picture

* Added Jason Hibbets photo

* Added Jason and Mel as RDU organizers.

* Update 2024-raleigh.yml

# out Propose

* Removed Alex from organizers

* Added emeritus to Craig Cook

He passed away about 6 months ago. We don't want to remove him from organizers.

terraboops committed:
[VAN-2024] Initial PR for Vancouver 2024 (#13556)

issyl0 committed:
[LDN-2024] Add Jen Mackown as a new organizer (#13557)

- I sent an email to core@ too.

strund3r committed:
update registration url (#13558)

mabaums committed:
[HOU-2024] Initial generation and adding organisers. (#13555)

* Initial set up with organizers for 2024 Houston

* removing donor placeholder

jerdog committed:
[KC-2024] misc updates (#13559)

* add missing square logo

* update Erin Crise info

* adjust sponsor level names

mabaums committed:
Add David

mabaums committed:
Adding Alex Zhanov

deniger committed:
[GE-2024] update welcome pages and add sponsors (#13560)

* GE-2024 logo with better quality and add sponsors

* GE-2024 logo with better quality and add sponsors

stealthybox committed:
[DEN-2024] Add Pretix Register Link (#13562)

fabioqcorreia committed:
[REC-2024] Initial Recife 2024 event (#13539)

romanoroth committed:
[ZRH-2024] Changed the URL of ZKB to the URL of the IT of ZKB] (#13564)

irabinovitch committed:
[LA 2024] CFP has been closed since Nov 1 (#13565)

[LA 2024] CFP has been closed since Nov 1

nimbinatus committed:
[ATX-2024] Weekly update (#13567)

* feat(sponsor): add txmutual

* fix(welcome): update welcome page with theme and links

deniger committed:
[GE-2024] fix link in the welcome page and remove propose (#13568)

* GE-2024 logo with better quality and add sponsors

* GE-2024 logo with better quality and add sponsors

* GE-2024 remove propose page

mabaums committed:
[HOU-2024] Add David and Alex as organizers (#13561)

* Add David

* Adding Alex Zhanov

stealthybox committed:
[DEN-2024] Add Pax8 + Parry Labs as Supporter sponsors (#13563)

* [DEN-2024] Add Pax8 as Supporter sponsor

* [DEN-2024] Add Parry Labs as Supporter sponsor

* [DEN-2024] Update Leigh's Employer

ivellios committed:
[KRK-2024] Krakow 2024 - Initial PR (#13541)

* Added initial files for Kraków 2024 event edition

* added description and photo for the organizer

* Update data/events/2024-krakow.yml

Co-authored-by: Adrian Moisey <adrian@changeover.za.net>

* replaced organizer image with a smaller one

* Updated links; fixed redirect

* added Call for Proposals link to navigation

---------

Co-authored-by: Katie McLaughlin <katie@devopsdays.org>
Co-authored-by: Adrian Moisey <adrian@changeover.za.net>

FloorD committed:
Add first sponsors, propose nav

DaniloBNascimento committed:
updating organizer profile Paulo Sizino

FloorD committed:
Add devleaps png logo

FloorD committed:
Merge pull request #13569 from devopsdays/sponsorsams24

[AMS-24] Add first sponsors, propose nav

jerdog committed:
add new organizer Jenny Tarwater

jerdog committed:
add twitter and image

glasnt committed:
Merge branch 'main' into don/archive-assets

glasnt committed:
Merge pull request #13573 from jerdog/jerdog/kc2024-addJenny

[KC-2024] add new organizer

glasnt committed:
Merge branch 'main' into don/archive-assets

glasnt committed:
Merge pull request #13018 from don-code/don/archive-assets

Move archived assets to CDN

glasnt committed:
Merge pull request #13570 from DodAracaju/Aju-2024/update_organizer_profile

[AJU-2024] Updating organizer profile Paulo Sizino

FloorD committed:
Add Schuberg sponsor

FloorD committed:
Merge pull request #13574 from devopsdays/amsschuberg

[AMS-24] Add Schuberg sponsor

jerdog committed:
[KC-2024] set registration to go liive (#13575)

* set registration to go live on 15-Jan-2024 at 8a local

* fix link to reg page

* fix test date

romanoroth committed:
[ZRH-2024] Fix the CFP announcement date (#13576)

omargohan committed:
[LA-2024] Updating schedule and speakers (#13572)

mpinedag12 committed:
[MDE-2024] Updating Dates and Message for Event (#13571)

* [MDE-2424] Updating Event Dates

[MDE-2424] Updating Event Dates

* [MDE-2024] Update Welcomen Message in welcome.md

[MDE-2024] Update Welcomen Message in welcome.md

---------

Co-authored-by: Katie McLaughlin <katie@devopsdays.org>

alvo254 committed:
[NRB-2024] NAIROBI 2024 - INITIAL PR (#13542)

* Adding new event, Nairobi devopsdays

* [Added new event for devopsdays Nairobi 2024 with organizers details]

* [Added new event for devopsdays Nairobi 2024 with organizers details]

* [Added new event for devopsdays Nairobi 2024 with organizers details]

* [Added new event for devopsdays Nairobi 2024 with organizers details]

* [changed 2024-nairobi.yaml]

* [time zone fix in 2024-nairobi.yaml]

* [team members 2024-nairobi.yaml]

* [team members image]

* [team members 2024-nairobi.yaml && image addition]

terraboops committed:
[VAN-2024] Add CFP details (#13578)

We are using a Google Form to collect CFP submissions, as that worked quite well for us in 2021.

dewan-ahmed committed:
Open CfP for Halifax-2024

dewan-ahmed committed:
Open CfP for Halifax-2024 (#13580)

themightymuppet committed:
update sponsor logo, dates, and CFP

glasnt committed:
Merge pull request #13581 from DevOpsDays-Denver/DEN-2024/sponsor-and-cfp

[DEN-2024] update sponsor logo, dates, and CFP

phrawzty committed:
Clarify when and why to use change_sponsor_logo.sh (#13582)

mkoperator committed:
[BHM-AL-2024] Add lynker sponsor. (#13583)

Co-authored-by: Mikhail Kozorovitskiy <misha@mkoperator.com>

phrawzty committed:
Clarify Matty and Yvo's terms as co-chairs

yvovandoorn committed:
Merge pull request #13586 from phrawzty/matty_n_yvo_tenure

Clarify Matty and Yvo's terms as co-chairs

frozenprocess committed:
[VAN-2024] CFP Page  (#13584)

* [VAN-2024] This PR makes the Vancouver CFP process a bit more discoverable.

* typo, spell

omargohan committed:
[LA-2024] Adding sponsor and updating logo (#13577)

**Summary of changes:**

- Adding SigLens as `gold` sponsor
- Updating Doppler logo as requested by sponsor

**Tested:**

- Locally using Hugo server

fabioqcorreia committed:
[REC-2024] Updating redirect for /recife and including /recife/cfp for call for papers link (#13579)

phrawzty committed:
[REC-2024] fix redirect (oops) (#13587)

phrawzty committed:
[minor] Sorting the contents of core.toml (#13588)

* Sorting the contents of core.toml

* fix missing comma

GeoffCraig committed:
[NSH-2024] Adds dates and sponsor info (#13585)

* Adds 2024 event information

* Updating sponsor page

* Adds logos
Updates sponsor page

* changes fontawesome to check

* changes fa icos to html checkboxes

* Updates location page

* Removes unneeded files

romanoroth committed:
[ZRH-2024] Remove Franziska Buehlker and Add Jan Moser (#13590)

* [ZRH-2024] Remove Franziska Buehlker and Add Jan Moser

* [ZRH-2024] Fixed file name

mattstratton committed:
[CHI-2024] Add Richard Lewis as organizer (#13591)

Signed-off-by: Matty Stratton <matty.stratton@aiven.io>

mauriciopgomes committed:
[RIO-2024] Inicial Commit

mauriciopgomes committed:
[RIO-2024] Inicial Commit

mauriciopgomes committed:
[RIO-2024] Inicial Commit

mauriciopgomes committed:
[RIO-2024] Inicial Commit

mauriciopgomes committed:
[RIO-2024] Inicial Commit

mauriciopgomes committed:
[RIO-2024] Inicial Commit

mauriciopgomes committed:
[RIO-2024] Inicial Commit

mauriciopgomes committed:
[RIO-2024] Inicial Commit

mauriciopgomes committed:
[RIO-2024] Inicial Commit

mauriciopgomes committed:
[RIO-2024] Inicial Commit

mauriciopgomes committed:
[RIO-2024] Inicial Commit

mauriciopgomes committed:
[RIO-2024] Inicial Commit

mauriciopgomes committed:
[RIO-2024] Inicial Commit

mauriciopgomes committed:
[RIO-2024] Inicial Commit

mauriciopgomes committed:
[RIO-2024] Inicial Commit

mauriciopgomes committed:
[RIO-2024] Inicial Commit

omargohan committed:
[LA-2024] Minor changes to homepage (#13593)

**Summary of changes:**
- Removing CFP from homepage
- Promoting the schedule in the homepage
**Tested:**
- Locally using hugo server

mabaums committed:
Event info update and organizer pictures upload

mabaums committed:
Event info update and organizer pictures upload

mauriciopgomes committed:
Merge branch 'main' into main

sebagomez committed:
Extend CFP one week

oseemann committed:
[BER-2024] Add Snyk as platinum sponsor

oseemann committed:
[BER-2024] Add Kubeevents as a partner sponsor

omargohan committed:
[LA-2024] Adding DNSimple as gold sponsor

**Sumamry of changes:**

- Adding DNSimple as `gold` sponsor

**Tested:**

- Locally using hugo server

davenash-lytx committed:
- Close CFP
- Add link for ticket sales.

glasnt committed:
Merge pull request #13594 from sebagomez/montreal-2024

[MTL-2024] Extend CFP one more week

glasnt committed:
Merge pull request #13596 from omargohan/feature/new_gold_sponsor_dns

[LA-2024] Adding DNSimple as gold sponsor

glasnt committed:
Merge pull request #13597 from oseemann/ber24-sponsors

[BER-2024] Two more sponsors

glasnt committed:
Merge pull request #13598 from harbinger55/sea24_tickets_cfp

[SEA-2024] CFP close and open tickets

mauriciopgomes committed:
fix change name

mauriciopgomes committed:
fix change name

mauriciopgomes committed:
Merge branch 'devopsdays:main' into main

mauriciopgomes committed:
fix change name

ih-cweeks committed:
Add sponsor prospectus link

FloorD committed:
[AMS-24] Add Elastic gold sponsor

FloorD committed:
Merge pull request #13603 from devopsdays/ams-elastic

[AMS-24] Add Elastic gold sponsor

baldwinjim committed:
[DFW-2024] - Initial setup for 2024 conference (#13599)

* initial commit

* initial commit

* initial updates for 2024 conference

* comment out links code that lead to 404s

---------

Co-authored-by: Jim Baldwin <jim.baldwin@wnco.com>

mabaums committed:
Location, and welcome page updates, David Varela jpg

mabaums committed:
Location, and welcome page updates, David Varela jpg

mabaums committed:
Merge branch 'add_david' of https://github.com/mabaums/devopsdays-web into add_david

glasnt committed:
Merge pull request #13602 from Clait/slc-2024-sponsor-updates

[SLC-2024] Add sponsor prospectus link

glasnt committed:
Merge pull request #13605 from mabaums/add_david

[HOU-2024] Updating Welcome page with Event info

phrawzty committed:
Clarify "core" vs "global core" on About page (#13601)

deniger committed:
[GE-2024] add program and update sponsors (#13600)

* GE-2024 remove propose page

* GE-2024 program and speakers

* GE-2024 program and speakers updates

mauriciopgomes committed:
Merge branch 'main' into main

baldwinjim committed:
[DFW-2024] updates to organizers, remove mike and megan (#13606)

mabaums committed:
Jim wang

stealthybox committed:
[DEN-2024] Upsert Observe Inc as sponsor

stealthybox committed:
[DEN-2024] Increase max supporter sponsors

glasnt committed:
Merge pull request #13607 from mabaums/add_david

[HOU-2024] Add Jim wang

glasnt committed:
Merge pull request #13608 from stealthybox/sponsors

[DEN-2024] Update Sponsors

deniger committed:
GE-2024 program and speakers updates (#13609)

mauriciopgomes committed:
Merge branch 'devopsdays:main' into main

mauriciopgomes committed:
Update _redirects

mauriciopgomes committed:
Update conduct-fr.md

mauriciopgomes committed:
Update conduct-pt-br.md

mauriciopgomes committed:
Update conduct.md

mauriciopgomes committed:
Update contact.md

mauriciopgomes committed:
Update program.md

mauriciopgomes committed:
Update propose.md

mauriciopgomes committed:
Update registration.md

mauriciopgomes committed:
Update 2024-rio-de-janeiro.yml

mauriciopgomes committed:
Update 2024-rio-de-janeiro.yml

mauriciopgomes committed:
Delete content/events/2024-rio-de-janeiro/speakers/modelo.md

mauriciopgomes committed:
Delete content/events/2024-rio-de-janeiro/program/modelo.md

joostpolley committed:
[MTL-2024] reformat sponsor html table

joostpolley committed:
[MTL-2024] remove lanyard sponsor type

joostpolley committed:
[MTL-2024] add sustainable swag sponsor type

sebagomez committed:
[MTL-2024] New sponsor (#13610)

* New sponsor

* fixed casing

mauriciopgomes committed:
Merge branch 'main' into main

gallimorej committed:
[DC-2024] Initialize the Washington DC 2024 event (#13604)

* Create all the things for the DevOpsDays DC 2024 event

* Remove organizer photos from last year

* Update organizers to match on website and email list

FloorD committed:
[AMS-24] Add sysdig silver, update orgs

FloorD committed:
Merge pull request #13611 from devopsdays/ams-sysdig

[AMS-24] Add sysdig silver, update orgs

DaniloBNascimento committed:
adding cfp link and date of start and end

mauriciopgomes committed:
Merge branch 'main' into main

joostpolley committed:
[MTL-2024] adapt prices for lunch, cocktail & coffee

joostpolley committed:
[MTL-2024] move sustainable giveaway to column 3

joostpolley committed:
[MTL-2024] blurb in comms for everyone

omargohan committed:
[LA-2024] Fixing schedule link (#13613)

**Summary of changes:**

- Fixing typo in schedule link

**Tested:**

- Locally using Hugo server

romanoroth committed:
[ZRH-2024 Add new sponsor VSHN] (#13614)

glasnt committed:
Merge pull request #13612 from DodAracaju/Aju-2024/add_cfp_link_and_date

[AJU-2024]  Adding cfp link and date of start and end

glasnt committed:
Merge pull request #13616 from polleydium/dodmtl-sponsorship-updates

[MTL-2024] update sponsership tarifs

joostpolley committed:
[MTL-2024] fix swag & platinum price swap (#13617)

strund3r committed:
[SP-2024] Update sponsorship info (#13566)

* update sponsorship info

* remove link to pdf

glasnt committed:
Merge branch 'main' into main

glasnt committed:
Merge pull request #13592 from mauriciopgomes/main

[RIO-2024] New Event 2024

glasnt committed:
Update core.toml (#13619)

Yvo has advised he is not advisory.

jrn00 committed:
added vets in devops logo

jrn00 committed:
added triangle tech talks logo

jrn00 committed:
added kubehuddle logo

jrn00 committed:
Create vets-in-devops.yml

jrn00 committed:
Create triangle-tech-talks.yml

jrn00 committed:
Create kubehuddle.yml

jrn00 committed:
Update 2024-raleigh.yml

added kubehuddle, vets in devops, and triangle tech talks as community sponsors

jrn00 committed:
Update 2024-raleigh.yml

baldwinjim committed:
conference date change

frozenprocess committed:
Sponsor page and general logo

glasnt committed:
Merge pull request #13620 from jrn00/main

[RDU-2024] Added Community Sponsors to Raleigh

glasnt committed:
Merge pull request #13622 from frozenprocess/van_sponsor_update

[VAN-2024] Sponsorship details and general logo update

glasnt committed:
Merge pull request #13621 from baldwinjim/2024--date-change

[DFW-2024] conference date change

deniger committed:
[GE-2024] add Open Space Opening on second day

FloorD committed:
Merge pull request #13623 from deniger/fix-ge

[GE-2024] add Open Space Opening on second day

frozenprocess committed:
[VAN-2024] More details about the packages. (#13625)

omargohan committed:
[LA-2024] Adding two more sponsors (#13626)

**Summary of changes:**

- Adding SauceLabs as Gold sponsor
- Adding Couchbase as Silver sponsor

**Tested:**

- Locally

DaniloBNascimento committed:
fix main aracaju page event redirect

ih-cweeks committed:
[SLC-2024] Postponing event, new dates unknown.

glasnt committed:
Merge pull request #13627 from DodAracaju/Aju-2024/fix_main_page_event_redirect

[AJU-2024] Fix main aracaju page event redirect

nimbinatus committed:
feat(sponsors): add Tailscale and Civo

glasnt committed:
Merge pull request #13628 from Clait/slc-2024-sponsor-updates

[SLC-2024] Postponing event, new dates unknown.

bridgetkromhout committed:
Merge pull request #13631 from anfleene/msp-2024-updates

[MSP-2024] Add 2024 Minneapolis sponsorship prospectus, Open CFP, and Ticket Sales

romanoroth committed:
[ZRH-2024] Add Digicomp as sponsor

romanoroth committed:
[ZRH-2024] Add Purestorage as sponsor

romanoroth committed:
[ZRH-2024] Add Cognizant as sponsor

romanoroth committed:
[ZRH-2024] Replace Ciklum logo with new logo

glasnt committed:
Merge pull request #13629 from nimbinatus/atx-2feb

[ATX-2024] 2 Feb Updates: Add Tailscale and Civo

glasnt committed:
Merge pull request #13632 from romanoroth/ZRH2024Week6Changes

[ZRH-2024] Week 06 Changes

glasnt committed:
Merge pull request #13633 from romanoroth/ZRH2024ReplaceCiklumLogo

[ZRH-2024] Replace Ciklum logo with new logo

Commit:
[MSP] Add Health and Safety page

bridgetkromhout committed:
Merge pull request #13635 from anfleene/msp-2024-health-and-safety

[MSP] Add Health and Safety page

Commit:
[MSP} tweak welcome page health and safety wording

bridgetkromhout committed:
Merge pull request #13636 from anfleene/welcom-page-update

[MSP] tweak welcome page health and safety wording

Commit:
Fix typo

bridgetkromhout committed:
Merge pull request #13637 from anfleene/fix-typo

[MSP] Fix typo with health and safety

sergiubodiu committed:
SIN-24 program update add/remove sponsors

yvovandoorn committed:
Merge pull request #13638 from devopsdays-singapore/singapore-program

SIN-24 program update add/remove sponsors

bushong1 committed:
[BMORE-2024] Closing Proposals, Opening tickets (#13634)

* [BMORE-2024] Closing Proposals, Opening tickets

* Fix missing div on welcome page

* Fix registration links

josvanschouten committed:
cfp is now closed (#13642)

tweinmann committed:
[ZRH-2024] changed cloudscale logo (#13640)

mabaums committed:
[HOU-2024] CFP and logo updates (#13641)

* Propose and logo updates

* Event info update and organizer pictures upload

* Location, and welcome page updates, David Varela jpg

* Jim wang

* Propose and logo updates

* CFP updates

* Event info update and organizer pictures upload

* Location, and welcome page updates, David Varela jpg

phrawzty committed:
[LON-2024] redirect /london to the 2024 event (#13644)

mattstratton committed:
Add description check to PRs (#13643)

* Add description check to PRs

Signed-off-by: Matty Stratton <matty.stratton@aiven.io>

* Add step to comment on PR

Signed-off-by: Matty Stratton <matty.stratton@aiven.io>

* Update pr.yml

* Add different message

Co-authored-by: Daniel Maher <phrawzty+github@gmail.com>

---------

Signed-off-by: Matty Stratton <matty.stratton@aiven.io>
Co-authored-by: Daniel Maher <phrawzty+github@gmail.com>

nimbinatus committed:
feat(tickets): open registration

nimbinatus committed:
fix(time): update to make live

nimbinatus committed:
fix(link): forgot to turn one on

nimbinatus committed:
fix(link): forgot to turn another on

jrn00 committed:
Added Observe as Gold Sponsor

bridgetkromhout committed:
Merge pull request #13645 from nimbinatus/atx-tickets

[ATX-2024] Open registration

strund3r committed:
update event venue

strund3r committed:
update ticket link

strund3r committed:
add link to sponsorship pdf

strund3r committed:
update sponsorship pricing in usd

strund3r committed:
fix typo

gomex committed:
Merge pull request #13647 from strund3r/sp-2024-update-info

[SP-2024] Update event info

FloorD committed:
Merge pull request #13646 from jrn00/main

[rdu-2024] Added Observe as Gold Sponsor

FloorD committed:
Add gold sponsor devoteam

FloorD committed:
Merge pull request #13648 from devopsdays/ams-devoteam

Add gold sponsor devoteam

deniger committed:
GE-2024 add 2 sponsors

tweinmann committed:
[ZRH-2024] go-live of program (#13615)

* [ZRH-2024] initial program

* [ZRH-2024] hide program & speakers for the time being

* [ZRH-2024] updated program & twitter links

* [ZRH-2024] added descriptions for Manuel Pais & changed order of some items

* [ZRH-2024] go-live of program

glasnt committed:
Merge pull request #13649 from deniger/ge-new-sponsor

[GE-2024] add 2 sponsors

mattstratton committed:
Update pr.yml

mattstratton committed:
Update pr.yml

mauriciopgomes committed:
[RIO-2024] update event location (#13652)

nimbinatus committed:
feat(sponsor): add RackN as gold (#13654)

skryukova committed:
[HOU-2024] Adding CFP announce date (#13656)

deniger committed:
[GE-2024] change bronze sponsor price to 1500 (#13659)

* [GE-2024] change bronze sponsor price to 1500

* Update les-renverses.yml

toshywoshy committed:
[AMS-2024] Add Mona to the team (#13657)

Signed-off-by: Toshaan Bharvani <toshaan@vantosh.com>

shanedroid committed:
[RDU-24] Add speakers, talks, and initial program  (#13661)

* chore: lexi-sort sponsors by id

* chore: added speakers for rdu-2024.

* chore: added talks for rdu-2024.

* chore: updated program for rdu-2024.

* chore: removed extra images not needed for rdu-24.

* chore: update more speaker photos for rdu-24.

turbut committed:
[VLN-2024] add event stub (#13662)

Co-authored-by: Ignas Poskus <ignas.poskus@devbridge.com>

phrawzty committed:
[core] Highliht Pretalx and Pretix

phrawzty committed:
[core] Remove misplaced directory (#13664)

phrawzty committed:
Update content/page/organizing.md

phrawzty committed:
Update content/page/organizing.md

phrawzty committed:
Merge branch 'main' into pretalx_and_tix

yvovandoorn committed:
Merge pull request #13665 from phrawzty/pretalx_and_tix

mattstratton committed:
[CHI-2024] Add Schuyler to organizers (#13666)

* [CHI-2024] Add Schuyler

Signed-off-by: Matty Stratton <matty.stratton@aiven.io>

* Update 2024-chicago.yml

---------

Signed-off-by: Matty Stratton <matty.stratton@aiven.io>

mesmith027 committed:
Adding Kube as community sponsors

yvovandoorn committed:
[GHENT-2024] Initial commit for Ghent (Antwerp) 2024 & Organiser Summit 2024 (#13630)

* add initial commit for DevOpsDays Ghent 2024 and the organiser day, securing the dates

* Update 2024-ghent.yml

fix date

* Update 2024-organiser-summit.yml

fix date

* Update _redirects

Added some edge cases for Ghent

* Add Ringo

* Update 2024-ghent.yml

updated to reflect Antwerp

* Update 2024-ghent.yml

finalized organizers

---------

Co-authored-by: Kris Buytaert <github@krisbuytaert.be>

turbut committed:
add sponsor (#13668)

Co-authored-by: Ignas Poskus <ignas.poskus@devbridge.com>

FloorD committed:
[AMS-24] Appcircle Gold sponsor

FloorD committed:
Merge pull request #13671 from devopsdays/ams-appcircle

[AMS-24] Appcircle Gold sponsor

mesmith027 committed:
Merge branch 'main' into main

KrisBuytaert committed:
Updating the organisers social media (#13672)

Co-authored-by: Kris Buytaert <sdog@inuits.eu>

KrisBuytaert committed:
Enhancing the description of the event (#13670)

Co-authored-by: Kris Buytaert <sdog@inuits.eu>

mattstratton committed:
[2024-ORG] Fix path (#13673)

delgod committed:
[KYIV-2024] add Kyiv 2024 (#13669)

glasnt committed:
Merge pull request #13667 from dewan-ahmed/main

[2024-Halifax] New Community Sponsor

omargohan committed:
[LA-2024] Adding everbridge as gold sponsor (#13674)

**Summary of changes:**

- Adding everbridge as gold sponsor

**Tested:**

- Locally using hugo server

sergiubodiu committed:
SIN-24 program update add/remove sponsors (#13676)

nimbinatus committed:
patch(pdf): change asset location

FloorD committed:
Merge pull request #13677 from nimbinatus/atx-16feb

[ATX-2024] 16 Feb 2024 update

omargohan committed:
[LA-2024] Updating everbridge logo (#13679)

**Summary of changes:**

- Updating everbridge's logo

**Tested:**

- Locally using hugo server

terraboops committed:
[VAN-2024] Extend CFP by a bit to get more lightning talks (#13680)

* [VAN-2024] Extend CFP by a week to get more lightning talks

* [VAN-2024] Extend it a smidge longer

mattstratton committed:
Fix GitHub actions permissions

mattstratton committed:
Merge branch 'main' into large-file-action

mattstratton committed:
fit yaml

mattstratton committed:
Update to fix GH action

mattstratton committed:
Delete .github/scripts/comment-pr-large-file.js

script no longer needed

mattstratton committed:
Update large-files.yml

fixed trigger
